src/mudlet-lua/lua/generic-mapper/generic_mapper.xml

```diff
@@ -2736,7 +2762,14 @@ local function handle_exits(exits)
     end
 end
 
-local continue_walk, timerID
+-- 定制开门指令可能有多条，以;分隔
```

Comment:
EN: `There may be multiple custom opening instructions, separated by;`

Reply:
fixed it

Reply:
en_GB: Okay, I just wanted to avoid non-English comments being in the source code - we still have some of Heiko's (I guess) German in some places...!
hz_CN: 好的，我只是想避免在源代码中包含非英语注释-在某些地方，我们仍然有一些Heiko（我想）德语...！

Reply:
> we still have some German in some places...!
Please open a new issue with more details so these can be checked and replaced

Reply:
PR #4276 for German comment removal created.